pyproject.toml

```diff
@@ -72,3 +72,64 @@ addopts = '''
     --ignore='yt/frontends/owls_subfind/tests/test_outputs.py'
     --ignore='yt/frontends/ramses/tests/test_outputs.py'
 '''
+
+
+[tool.check-manifest]
```

Comment:
This does add another place that we include all the names of the generated cython files, which I'm a little hesitant about.

Reply:
Yeah, not a fan either. If you think the cost isn't worth the time this supposedly saves on releases, maybe we can just drop it.